public/spec/graphql/spec.md

```diff
@@ -821,11 +821,11 @@ GraphQL `Union` type represent an object that could be one of a list of GraphQL
 > **Note:** Only `distinct` service types are supported as members of a union type in the Ballerina GraphQL package. If one or more members in a union type do not follow this rule, a compilation error will be thrown.
 
 ###### Example: Union Types
-In the following example, two `distinct` service types are defined first, `Teacher` and `Student`. Then a `Union` type is defined using Ballerina syntax for defining union types. The resource function in the GraphQL service is returning the union type.
+In the following example, two `distinct` service types are defined first, `Teacher` and `Student`. Then a `Union` type is defined using Ballerina syntax for defining union types. The resource method in the GraphQL service is returning the union type.
```

Comment:
Only changes to text such as this is relevant.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6013.